x-pack/plugins/stack_connectors/common/opsgenie/index.ts

```diff
@@ -10,4 +10,6 @@ export enum OpsgenieSubActions {
   CloseAlert = 'closeAlert',
 }
 
+export const RULE_TAGS_TEMPLATE = `{{rule.tags}}`;
```

Comment:
This template is defined in the `triggers_actions_ui` plugin but it is not exported and it is part of an enum so it'd be kind of difficult to refactor. I'm duplicating it here for simplicity.